docs/usage.md

```diff
@@ -89,27 +89,23 @@ These are used to set display properties for a specific dataset.
 | ---- | ---- | :----: | ----
 | [`backgroundColor`](#styling) | [`Color`](https://www.chartjs.org/docs/latest/general/colors.html) | Yes | `undefined`
 | [`borderColor`](#styling) | [`Color`](https://www.chartjs.org/docs/latest/general/colors.html) | Yes | `undefined`
-| [`borderWidth`](#styling) | `number`\|`object` | Yes | `0`
-| [`color`](#styling) | [`Color`](https://www.chartjs.org/docs/latest/general/colors.html) | Yes | `undefined`
-| [`dividerCapStyle`](#divider) | `string` | Yes | `'butt'`
-| [`dividerColor`](#divider) | [`Color`](https://www.chartjs.org/docs/latest/general/colors.html) | Yes | `'black'`
-| [`dividerDash`](#divider) | `number[]` | Yes | `undefined`
-| [`dividerDashOffset`](#divider) | `number` | Yes | `0`
-| [`dividerWidth`](#divider) | `number` | Yes | `1`
-| [`font`](#styling) | `Font` | Yes | `{}` 
-| [`groupDividers`](#general) | `boolean` | Yes | `false` |
-| [`groupLabels`](#general) | `boolean` | Yes | `true` |
+| [`borderWidth`](#styling) | `number`\|`object` | - | `0`
+| [`captions`](#captions) | `object` | - | 
+| [`dividerCapStyle`](#divider) | `string` | - | `'butt'`
+| [`dividerColor`](#divider) | [`Color`](https://www.chartjs.org/docs/latest/general/colors.html) | - | `'black'`
+| [`dividerDash`](#divider) | `number[]` | - | `undefined`
+| [`dividerDashOffset`](#divider) | `number` | - | `0`
+| [`dividerWidth`](#divider) | `number` | - | `1`
+| [`groupDividers`](#general) | `boolean` | - | `false` |
```

Comment:
Just a note to self, should scope these under `dividers` :)

Reply:
@kurkle  yes!! maybe you don't believe me but it is in my TODO list but if you want to do it, more than happy!!!

Reply:
I'll get to it eventually. Currently too busy with other stuff though. So I'll welcome a PR too :)

Reply:
> I'll get to it eventually. Currently too busy with other stuff though. So I'll welcome a PR too :)

I'm busy too (I'm using my spare time) but I'll do my best to submit a PR, sooner or later.

Reply:
> 
> 
> Just a note to self, should scope these under `dividers` :)

PR #55 ;)